Sources/Commands/Workspace.swift

```diff
@@ -83,20 +87,27 @@ public class Workspace {
         /// The current version of the dependency, if known.
         public let currentVersion: Version?
 
+        /// The dependency is being edited.
+        let isEdited: Bool
+
         /// The current revision of the dependency.
         ///
         /// This should always be a revision corresponding to the version in the
         /// repository, but in certain circumstances it may not be the *current*
         /// one (e.g., if this data is accessed with a different version of the
         /// package manager, which would cause an alternate version to be
         /// resolved).
+        /// For a dependency in edit mode, this will the version at which the
```

Comment:
I think this should clarify what "at which the dependency was checked out" means (given the overloading of the term "checked out")... I think it means it will be the version at which the dependencies was made editable?